src/core/utils/index.js

```diff
@@ -336,3 +348,39 @@ export const getAddonTypeFilter = (addonType, _config = config) => {
 
   return ADDON_TYPE_THEMES_FILTER;
 };
+
+export const getExtensionTypeAction = (type = ADDON_TYPE_EXTENSION) => {
+  let action = TRACKING_TYPE_EXTENSION;
+
+  if (type === ADDON_TYPE_STATIC_THEME) {
+    action = TRACKING_TYPE_STATIC_THEME;
+  }
+
+  return action;
+};
+
+export const getAddonCategory = ({
```

Comment:
`getAddonCategory` seems a bit of a vague name for this function, as it's not really the category of an add-on. Perhaps `getAddonEventCategory` would be better?